tokens.json

```diff
@@ -1210,5 +1210,16 @@
       "twitter": "https://twitter.com/DigitalGold2022",
       "discord": "https://discord.gg/QSvKpmeRjZ"
     }
+    },
+  "df1d850c46d6c9d12cbf6181c35db9225a91b77c8a646b7f636f8ae4": {
+    "project": "Danketsu Token $NINJAZ",
+    "categories": ["GameFi, "NFT", "Governance"],
```

Comment:
@vivid4516 you missed a double quote here. please update this PR asap

Reply:
Fixed